src/sonic-yang-models/tests/yang_model_tests/tests/acl.json

```diff
@@ -100,7 +100,14 @@
         "eStrKey" : "Pattern"
     },
     "ACL_RULE_L2_INVALID_ETHER": {
-        "desc": "Configure invalid MAC address format.",
```

Comment:
why do you remove the UT for invalid MAC address?

Reply:
I didn't remove any UT. I just corrected the description which was wrong.